src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return color.toString();
```

Comment:
To get the String representation of an enum constant, use the name() method instead of toString(). This ensures that you get the correct name even if the toString() method is overridden in the enum.